demo/game/index.js

```diff
@@ -80,18 +104,25 @@ export default class Game extends Component {
       fade: false,
     });
 
+    this.stageXUIUnsubscribe = GameStore.onStageXChange((stageX) => {
+      this.forceUpdate();
+    });
```

Comment:
As the stage is scrolled, we want to also move our debug canvas, so we force an update on change.